Samples/DecoderSample/Classes/LoraDecoders.cs

```diff
@@ -1,19 +1,23 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using System.Text;
 using Newtonsoft.Json;
 
 namespace SensorDecoderModule.Classes
 {
     internal static class LoraDecoders
     {   
-        private static string DecoderValueSensor(byte[] payload, uint fport)
+        private static string DecoderStringValueSensor(byte[] payload, uint fport)

```

Comment:
It is a bit tricky that we have in the NtwSrv the name DecoderValueSensor and here the same with another name. Also if you change it here the doc should reflect it. I would keep it DecoderValueSensor if not we need to change docs and the NtwSrv

Reply:
Good input. I keep one DecoderValueSensor and use code & comments inside to show how to decode both type of payloads...!

Reply:
Updated!